addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -53,7 +53,7 @@
                     'plus_report_line_ids': [ref('l10n_de.tax_report_de_tag_61')],
                 }),
             ]"/>
-            <field name="tax_group_id" ref="tax_group_0"/>
+            <field name="tax_group_id" ref="tax_group_19"/>
```

Comment:
Too bad this line isn't up there with the amount, it'd be easier to check ^^

But you can leave it like this since it's done everywhere in the file